src/include/sof/dai.h

```diff
@@ -66,16 +80,21 @@ struct dai_ops {
 	int (*set_loopback_mode)(struct dai *dai, uint32_t lbm);
 };
 
-/* DAI slot map to audio channel */
+/**
+ * \brief DAI slot map to audio channel
+ */
 struct dai_slot_map {
-	uint32_t channel;	/* channel ID - CHAN_ID_ */
-	uint32_t slot;		/* physical slot index */
+	uint32_t channel;	/**< channel ID - CHAN_ID_ */
+	uint32_t slot;		/**< physical slot index */
 };
 
+/**
+ * \brief DAI Type.
+ */
 enum dai_type {
```

Comment:
Unrelated to this patch: I never realized this, but there is already a list of DAI types in ipc.h, of course with different values. Can this be optimized or do we have a reason for the duplication?